operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -36,4 +65,10 @@ if [ -f ${STATEFILE} ] && [ `grep -c "FAILED_NOT_RESTARTABLE" ${STATEFILE}` -eq
   trace SEVERE "WebLogic Server state is FAILED_NOT_RESTARTABLE."
   exit $RETVAL
 fi
+
+newCopySitCfg /weblogic-operator/introspector ${DOMAIN_HOME}/optconfig             'Sit-Cfg-CFG--'
```

Comment:
It seems odd to me to do processing other than checking whether the container is running or not in the liveness probe, but I am by no means expert in this.

Reply:
This is a key part of the new functionality - the sit config files can be changed in the domain spec, which means that they need to be copied into their proper locations. If we roll the pods, that happens automatically, but the customer wants to be able to apply non-dynamic changes without rolling. The only way we could figure out to do that was by the liveness script, which is already running at intervals.

I agree that it seems clumsy and non-obvious. My original thought was that the operator could run a script, but it doesn't know when the config map with the files changed has actually happened. 

One possibility that we might try is having the operator run a script (via Kubernetes exec()) that does this testing and reports on whether the files have been changed. It could then decide whether it needs to do it again. That seems as though it might be cleaner, if it would work.

Reply:
I am just worried if there might be some undesirable side-effect of using the liveness probe script this way. 

Another possibility could be starting a script from startServer.sh that wakes up periodically to do this.